### PR TITLE
adding the abouts page

### PR DESCRIPTION
**Pull request by sculley**
## Describe your changes

Adding an abouts page as per request in #139 

## Issue ticket number and link
#139 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [x] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/143"></a>

### CODE REVIEW

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Looks almost perfect. Could you just rename the file and the component to remove the s at the end. It should be `about.tsx`

Also, the router is not needed in this file so the import and const router line can be removed

**Comment by sculley** [comment]
Sure will do, funnily enough I renamed from about to abouts just before committing 😀

**Comment by sculley** [comment]
@EmmaDawsonDev - I have changed that now, thanks!

File: pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import type { NextPage } from "next"
+import { useRouter } from "next/router"
```
Comment (EmmaDawsonDev) [inline pages/about.tsx:2]:
```suggestion

```

File: pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import Layout from "../components/Layout/Layout"
+import Head from "next/head"
+import WorkInProgress from "../components/WorkInProgress/WorkInProgress"
+
+const About: NextPage = () => {
+	const router = useRouter()
```
Comment (EmmaDawsonDev) [inline pages/about.tsx:8]:
```suggestion
	
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
thanks for updating the name :) Could you also remove the router from this file? It's not needed as we're not passing anything to active nav link.

File: pages/about.tsx (hunk 1 above)
Comment (sculley) [reply pages/about.tsx]: Removed in latest commit.

File: pages/about.tsx (hunk 2 above)
Comment (sculley) [reply pages/about.tsx]: Removed in latest commit.

**Review by EmmaDawsonDev — APPROVED** [review]
Looks great, thank you!

**Pull request merged**